README.md

```diff
@@ -424,6 +424,8 @@ Feel free to propose your own bot to our collection!
  - [msfjarvis/walls-bot-rs](https://github.com/msfjarvis/walls-bot-rs) -- Telegram bot for my wallpapers collection, in Rust.
  - [MustafaSalih1993/Miss-Vodka-Telegram-Bot](https://github.com/MustafaSalih1993/Miss-Vodka-Telegram-Bot) -- A telegram bot written in rust using "Teloxide" library.
  - [x13a/tg-prompt](https://github.com/x13a/tg-prompt) -- Telegram prompt.
+ - [magnickolas/remindee-bot](https://github.com/magnickolas/remindee-bot) -- Reminder bot for Telegram without bullshit.
+ - [perosar/perorustbot](https://github.com/perosar/perorustbot) -- My personal telegram bot in rust.
```

Comment:
What does "bulls hit" mean?

I don't think "my" here is appropriate.

Reply:
I just copied the project descriptions from their GitHubs.

Reply:
First description can be changed to `Telegram bot for managing reminders` as in the README.md

Reply:
Perorustbot seems to be just another bot that doesn't do anything serious (just check the available commands). I think there is no reason to publish it in our readme.